src/chat/UnreadNotice.js

```diff
@@ -29,23 +29,32 @@ const styles = StyleSheet.create({
   }
 });
 
-const POSITIONS = {
-  top: 'top',
-  bottom: 'bottom'
-};
-
 const showAnimationConfig = {
   toValue: 1,
   duration: 400,
-  useNativeDriver: true,
-  easing: Easing.bezier(0.17, 0.67, 0.11, 0.99)
+  easing: Easing.bezier(0.17, 0.67, 0.11, 0.99),
+  useNativeDriver: true
 };
 
 const hideAnimationConfig = {
   toValue: 0,
   duration: 400,
-  useNativeDriver: true,
-  easing: Easing.bezier(0.17, 0.67, 0.11, 0.99)
+  easing: Easing.bezier(0.17, 0.67, 0.11, 0.99),
+  useNativeDriver: true
+};
+
+// Duration after which notice should hide
+const HIDE_DELAY = 1500;
+// The translation of notice interpolates this.translateAnimation from 0 to 1
+// Fraction of interpolation at which notice goes into peeking state
+const PEEKING_FRACTION = 0.3;
+// Amount notice should translate to get itself into the screen
+const MAX_TRANSLATION = 40;
+
+const noticeStates = {
+  hidden: 0,
```

Comment:
Yes, this makes sense since UnreadNotice is the only subject STATE_HIDDEN would intuitively translate to noticeState.hidden 👍